tests/rules/experimental/rbe/network_disabled.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
add short doc with purpose of this script

Reply:
Done.